### ISSUE TITLE
Not working with electron 4.1.0 and node 10.15.3

### ISSUE THREAD

**Issue by lilliesAndRoses** [jongear/xpc-connect#5]
Hi,
I am trying to use this repo as the dependency of noble instead of the old xpc-connection. 
I am trying this on MacOS 10.14. I do not get any issue during npm install but I am getting following runtime error - 
The module ‘/path/node_modules/xpc-connect/build/Release/xpc-connection.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 64. This version of Node.js requires
NODE_MODULE_VERSION 69. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).

How should I get around this?

**Comment by jongear** [comment]
I actually ran into this exact same issue on Mojave. The short answer is [Noble](https://www.npmjs.com/package/noble) will try and use [xpc-connection](https://www.npmjs.com/package/xpc-connection) which is an outdated package that doesn't support Mojave's bindings. The only sane way I found around this was to use a secondary package called `noble-mac` to resolve the issue. Checkout the following link to see how I used this dependency in another project.

https://github.com/jongear/minidrone/blob/master/package.json#L120

Hope this helps :smile:
